README.md

```diff
@@ -41,9 +41,9 @@ In order to grade an assignment, one has to run the `submission/grade` task. But
 
 where `partIdOfGradingProject` is one of the assignment part ids used in the `projectDetailsMap` of the main `build.sbt` file. Defining the `partIdOfGradingProject` setting can also be done when running from the command line:
 
- > ``sbt `(partIdOfGradingProject in submissionProject) := "idString"` submission/grade``
+ > ``sbt 'set (partIdOfGradingProject in submissionProject) := "idString"' submission/grade``
 
-The `submission/grade` task will then unpack the source, compile it, compile the tests, run the tests, run the stylechecker, and upload the generated feedback via the coursera API.
+The `submission/grade` task will then unpack the source, compile it, compile the tests, run the tests, run the stylechecker, and output the feedback and grade to stderr.
```

Comment:
Within GrID, stderr is a debug log, and stdout should be a single JSON object with a `feedback` field (and a grading field). GrID stores stderr for debugging purposes (viewable by instructional staff), and stdout is parsed, validated, and the feedback is made available in the platform UI.

Note: above is reversed compared to the README. That's because the wrapper script swaps stdout and stderr. (Because sbt is pretty noisy, intended for interactive use.) I can talk though the security implications on the phone call.